routes/error-tracker.js

```diff
@@ -30,9 +30,7 @@ const CF_METADATA = {
   resource: {
     type: 'cloud_function',
     labels: {
-      service_name: process.env.K_SERVICE,
-      version_id: process.env.K_REVISION,
-      commit_sha: process.env.COMMIT_SHA,
+      function_name: process.env.K_SERVICE,
```

Comment:
Shouldn't we continue logging the rest?

Reply:
They get dropped. It's not entirely clear why since labels are supposed to allow arbitrary key-value pairs, and I plan to investigate so the logging can get more details, but for the time being I wanted to at least make sure the function name is set correctly so I can verify changes in Beta aren't breaking logging/reporting.